tests/e2e/common.go

```diff
@@ -571,3 +553,13 @@ func setupOutputDir() {
 	}
 
 }
+
+func CheckEventually(testData string, actualOrCtx interface{}) AsyncAssertion {
+	By(testData)
+	return Eventually(actualOrCtx, testTimeout, testPollingInterval)
+}
+
+func CheckConsistently(testData string, actualOrCtx interface{}) AsyncAssertion {
+	By(testData)
+	return Consistently(actualOrCtx, testTimeout, testPollingInterval)
+}
```

Comment:
I want to get this merged sooner than later so you don't have merge conflicts, but one thing I'm thinking is that it would be great to be able to optionally specify the timeout or the polling interval if we need to.

Reply:
oh wait, cool, I see you were able to do that by calling `CheckEventually(...).WithTimeout(...)`, right?

Reply:
maybe we can just comment these functions to clarify that you can override the default timeout and default poll interval that way?